XPlat/OfficeUIFabric/Android/source/Additions/Additions.cs

```diff
@@ -17,30 +17,25 @@ public partial class Spacing
                 // So we manually remove the method and implement it manually
 
                 // Metadata.xml XPath method reference: path="/api/package[@name='com.microsoft.officeuifabric.persona']/class[@name='PersonaView.Companion.Spacing']/method[@name='copy' and count(parameter)=2 and parameter[1][@type='int'] and parameter[2][@type='int']]"
-                [Register("copy", "(II)Lcom/microsoft/officeuifabric/persona/PersonaView$Companion$Spacing;", "")]
-                public unsafe global::Microsoft.OfficeUIFabric.PersonaView.PersonaViewCompanion.Spacing Copy(int cellPadding, int insetLeft)
-                {
-                    const string __id = "copy.(II)Lcom/microsoft/officeuifabric/persona/PersonaView$Companion$Spacing;";
-                    try
-                    {
-                        JniArgumentValue* __args = stackalloc JniArgumentValue[2];
-                        __args[0] = new JniArgumentValue(cellPadding);
-                        __args[1] = new JniArgumentValue(insetLeft);
-                        var __rm = _members.InstanceMethods.InvokeNonvirtualObjectMethod(__id, this, __args);
-                        return global::Java.Lang.Object.GetObject<global::Microsoft.OfficeUIFabric.PersonaView.PersonaViewCompanion.Spacing>(__rm.Handle, JniHandleOwnership.TransferLocalRef);
-                    }
-                    finally
-                    {
-                    }
-                }
+                //[Register("copy", "(II)Lcom/microsoft/officeuifabric/persona/PersonaView$Companion$Spacing;", "")]
+                //public unsafe global::Java.Interop.IJavaPeerable Copy(int cellPadding, int insetLeft)
+                //{
+                //    const string __id = "copy.(II)Lcom/microsoft/officeuifabric/persona/PersonaView$Companion$Spacing;";
+                //    try
+                //    {
+                //        JniArgumentValue* __args = stackalloc JniArgumentValue[2];
+                //        __args[0] = new JniArgumentValue(cellPadding);
+                //        __args[1] = new JniArgumentValue(insetLeft);
+                //        var __rm = _members.InstanceMethods.InvokeNonvirtualObjectMethod(__id, this, __args);
+                //        return global::Java.Lang.Object.GetObject<global::Java.Interop.IJavaPeerable>(__rm.Handle, JniHandleOwnership.TransferLocalRef);
+                //    }
+                //    finally
+                //    {
+                //    }
+                //}
```

Comment:
I would just delete the commented out code. Unless it has some significance?

```suggestion
```

Reply:
You would. I will.
You know me. I try to do as much as possible in metadata transforms and was not sure if that will work immediately, so I kept the source. (Not fond of reading through the git history)

Reply:
Besides. I need to check that sample